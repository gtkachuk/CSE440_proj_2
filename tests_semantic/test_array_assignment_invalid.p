program testArrayAssignmentInvalid;

class testArrayAssignmentInvalid

BEGIN
   
   VAR XRay : ARRAY[0 .. 4] OF integer;
       YRay : ARRAY[8 .. 13] OF integer;

FUNCTION testArrayAssignmentInvalid;
BEGIN
   XRay := YRay
END

END
.
