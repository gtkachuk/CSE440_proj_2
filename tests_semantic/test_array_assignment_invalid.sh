#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_array_assignment_invalid.p
