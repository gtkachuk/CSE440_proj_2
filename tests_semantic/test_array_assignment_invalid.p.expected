ERROR:13:Type mismatch between "__usrdef_0" and "__usrdef_1"
Errors detected. Exiting.
